modin/core/io/column_stores/parquet_dispatcher.py

```diff
@@ -609,18 +609,24 @@ def _read(cls, path, engine, columns, **kwargs):
         )
         from modin.pandas.io import PQ_INDEX_REGEX
 
-        if isinstance(path, str) and os.path.isdir(path):
+        if isinstance(path, str):
+            if os.path.isdir(path):
+                path_generator = os.walk(path)
+            else:
+                storage_options = kwargs.get("storage_options")
+                fs, fs_path = url_to_fs(path, **storage_options)
```

Comment:
`storage_options` variable can be `None`, so this construction breaks several tests in CI.

Reply:
Done.